scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/Job.java

```diff
@@ -440,7 +459,7 @@ public boolean isEqual(Job job) {
                finishTime == job.finishTime && inErrorTime == job.inErrorTime &&
                cumulatedCoreTime == job.cumulatedCoreTime && parentId == job.parentId &&
                childrenCount == job.childrenCount && numberOfNodes == job.numberOfNodes &&
-               numberOfNodesInParallel == job.numberOfNodesInParallel;
+               numberOfNodesInParallel == job.numberOfNodesInParallel && submissionMode.equals(job.submissionMode);
```

Comment:
can submissionMode be null ? if yes, then this test can trigger a NullPointerException